docs/en/guides/Plugin-test.md

```diff
@@ -294,6 +294,48 @@ The verify description for SegmentRef
 | networkAddress | The peer value of parent exit span.
 | refType | Ref type, options, CrossProcess or CrossThread.
 
+**Meter verify description format**
+```yml
+meterItems:
+-
+  serviceName: SERVICE_NAME(string)
+  meterSize: SEGMENT_SIZE(int)
+  meters:
+  - ...
+```
+
+| Field |  Description
+| --- | ---  
+| serviceName | Service Name.
+| meterSize | The number of meters is expected.
+| meters | meter list. Follow the next section to see how to describe every meter.
+
+**Meter data verify description format**
+
+```yml
+    meterId: 
+        name: NAME(string)
+        tags:
+        - {name: TAG_NAME(string), value: TAG_VALUE(string)}
+    singleValue: SINGLE_VALUE(double)
+    histogram:
+    - {bucket: HISTOGRAM_BUCKET(double), count: HISTOGRAM_BUCKET_COUNT(long)}
```

Comment:
How could you know the count? the values are generated in the runtime, I have doubt that it is possible to be accurate.